pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -81,7 +81,8 @@ def attach_data(self, model, train_dataloader, val_dataloaders, datamodule):
 
         # set up the passed in dataloaders (if needed)
         self.attach_dataloaders(model, train_dataloader, val_dataloaders)
-        self.attach_datamodule(model, datamodule, 'fit')
+        self.attach_datamodule(model, datamodule)
```

Comment:
Those parts also aren't relevant for this PR, right?

Reply:
already reverted.